### PR TITLE
Add private readonly IConfiguration variable

### PR DESCRIPTION
**Pull request by insukcho**
Add below private variable definition line in Startup.cs source snippet.

> private readonly IConfiguration Configuration;

### CODE REVIEW

File: docs/configuration.asciidoc [hunk 1]
```diff
@@ -33,7 +33,10 @@ using Elastic.Apm.AspNetCore;
 
 public class Startup
 {
-    public Startup(IConfiguration configuration)
+
+    private readonly IConfiguration Configuration;
```
Comment (russcam) [inline docs/configuration.asciidoc:37]: we should rename this from `Configuration` to `_configuration`, in line with private field naming conventions. Line 41 would also need to change to `_configuration = configuration;`

**Review by russcam — CHANGES_REQUESTED** [review]
Thanks for the submission, @insukcho - I left a comment about naming conventions.

File: docs/configuration.asciidoc (hunk 1 above)
Comment (insukcho) [reply docs/configuration.asciidoc]: agree. I made a new commit.

**Review by russcam — APPROVED** [review]
LGTM 👍

File: docs/configuration.asciidoc [hunk 2]
```diff
@@ -33,9 +33,12 @@ using Elastic.Apm.AspNetCore;
 
 public class Startup
 {
-    public Startup(IConfiguration configuration)
+
+    private readonly IConfiguration _configuration;
+    
+	public Startup(IConfiguration configuration)
```
Comment (russcam) [inline docs/configuration.asciidoc:39]:
```suggestion
    public Startup(IConfiguration configuration)
```

Comment (russcam) [reply docs/configuration.asciidoc:39]: minor nitpick: spacing

**Review by russcam — APPROVED** [review]

**Pull request merged**